lib/sovren/contact_information.rb

```diff
@@ -21,13 +21,39 @@ def self.parse(contact_information)
       result.postal_code = contact_information.css('PostalAddress').first.css('PostalCode').text rescue nil
       result.country = contact_information.css('PostalAddress').first.css('CountryCode').text rescue nil
 
-      result.home_phone = contact_information.css('Telephone FormattedNumber').first.text rescue nil
-      result.mobile_phone = contact_information.css('Mobile FormattedNumber').first.text rescue nil
+      result.phone_numbers = []
+      mobile_phones = contact_information.css('Mobile FormattedNumber').map { |m| { type: 'mobile', number: m.text } } rescue nil
+      fax_phones = contact_information.css('Fax FormattedNumber').map { |m| { type: 'fax', number: m.text } } rescue nil
 
-      result.websites = contact_information.css('InternetWebAddress').map { |w| w.text } rescue nil
-      result.email = contact_information.css('InternetEmailAddress').first.text rescue nil
+      other_phone_nodes = contact_information.css('ContactMethod')&.select { |node| node.css('Telephone').present? }
+      other_phones = parse_other_phone_types(other_phone_nodes) if other_phone_nodes.present?
+
+      result.phone_numbers.concat(mobile_phones) if mobile_phones.present?
+      result.phone_numbers.concat(fax_phones) if fax_phones.present?
+      result.phone_numbers.concat(other_phones) if other_phones.present?
```

Comment:
i guess it isn't necessary with these checks, would save us from needing these checks if they defaulted to an empty array though. doesn't matter either way